pages/price-feeds/best-practices.mdx

```diff
@@ -61,7 +64,7 @@ This analogy suggests two simple solutions to races:
 1. Configure protocol parameters to balance the losses from smart flow against the gains from two-way flow.
    Market makers in traditional finance implement this approach by offering a bid/ask spread and limited liquidity.
    The limited liquidity caps the losses to smart flow, while still earning profits from the two-way flow.
-   A successful market maker tunes the spread and offered liquidity to limit adverse selection from smart traders while still interacting with two-way flow.
+   A successful market maker tunes the spread and offers liquidity to limit adverse selection from smart traders while still interacting with the two-way flow.
```

Comment:
it's actually supposed to be "offered" here -- liquidity refers to the quantity of assets offered for sale at the price